lib/drivers/docs/tandem.md

```diff
@@ -174,14 +181,16 @@ No Tidepool data model yet:
     - `[ ]` extended bolus type (`manual` or `percentage`)
     - `[ ]` min BG to allow calculation of bolus delivery
     - `[ ]` reverse correction enabled
-    - `[ ]` max bolus
+    - `[-]` max bolus
     - "quick"/manual bolus:
-        - `[ ]` enabled
-        - `[ ]` increment
-  - `[ ]` insulin action time
+        - `[-]` enabled
+        - `[-]` increment
+  - `[-]` insulin action time
   - `[ ]` clock display preference (12h vs 24h format)
 
-Device-specific? (Add any device-specific notes/additions here.)
+##### Device-specific? (Add any device-specific notes/additions here.)
+
+- To build a full settings history we'll need to use the entire log record, not just the last 90 days. For example, a profile name change event only contains the new name, so you'll need to read the record where it was first created to get the original name. Until we do diff uploads (so it only happens once), or significantly improve the upload speed, reading the entire log record is not possible.
```

Comment:
Add a note maybe about how instead of 'bolus “wizard”/calculator enabled' Tandem has carbs enabled or not (which is not quite the same thing).